react/Badge/Badge.js

```diff
@@ -4,13 +4,14 @@ import React, { PureComponent } from 'react';
 import classnames from 'classnames';
 
 import styles from './Badge.less';
+import { TONE } from '../private/tone';
 
 type Tone =
   | 'accent'
-  | 'critical'
-  | 'info'
-  | 'neutral'
-  | 'positive'
+  | typeof TONE.CRITICAL
```

Comment:
If we start using it across multiple components probably it make sense to create a shared TONE type 🤔 Not necessarily in this PR, just thinking

Reply:
We have one but not all components accept the same values for tone https://github.com/seek-oss/seek-style-guide/blob/master/react/private/tone.js#L2